shared-helpers/src/locales/vi.json

```diff
@@ -28,6 +28,7 @@
   "account.settings.placeholders.month": "Tháng",
   "account.settings.placeholders.year": "Năm",
   "account.settings.update": "Cập nhật",
+  "alert.maintenace": "Trang web này đang được bảo trì theo lịch trình. Chúng tôi xin lỗi vì bất cứ sự bất tiện nào.",
```

Comment:
There is a typo in `alert.maintenance` causing this string to not get picked up for vietnamese